geoips/plugins/modules/output_checkers/netcdf.py

```diff
@@ -209,14 +229,13 @@ def outputs_match(plugin, output_product, compare_product):
             if meandiff != 0:
                 LOG.interactive(f"    *** meandiff {varname}: {meandiff} ***")
                 diffout += [f"meandiff {varname}: {meandiff}\n"]
-        LOG.info("    ****************************************************************")
+        LOG.info("    " + "*" * 64)
         retval = False
 
     try:
         xarray.testing.assert_identical(compare_xobj, out_xobj)
     except AssertionError as resp:
-        LOG.info("    ****************************************************************")
-        LOG.info("    *** INFORMATIONAL ONLY assert_identical differences *****")
+        log_with_emphasis(LOG.info, ["INFORMATIONAL ONLY assert_identical differences"])
```

Comment:
This is another one that should have multiple lines passed to a single "log_with_emphasis" call.